server/model/team.go

```diff
@@ -0,0 +1,45 @@
+package model
+
+// Team is information global to a team
+// swagger:model
+type Team struct {
+	// ID of the team
+	// required: true
+	ID string `json:"id"`
+
+	// Title of the team
+	// required: false
+	Title string `json:"title"`
+
+	// Token required to register new users
+	// required: true
+	SignupToken string `json:"signupToken"`
+
+	// Team settings
+	// required: false
+	Settings map[string]interface{} `json:"settings"`
+
+	// ID of user who last modified this
+	// required: true
+	ModifiedBy string `json:"modifiedBy"`
```

Comment:
So far we're inconsistent naming `ModifiedBy` vs `ModifiedByID` throughout the codebase.  Looking more closely I see my code is using `MofifiedByID` the rest is `ModifiedBy`.  :grimacing:

Reply:
I've based myself on the `blocks` table as the boards are coming from it, but happy to standardize in either direction really :)

Besides that, just saw that the `json` annotations are using camel case and not snake case, so I'll be changing that as part of this PR too :sweat_smile:

Reply:
Updated `json` annotations on e8be3ca